MetadataExtractor/Formats/QuickTime/QuickTimeMetadataReader.cs

```diff
@@ -62,7 +62,7 @@ void TrakHandler(AtomCallbackArgs a)
                         directory.Set(QuickTimeTrackHeaderDirectory.TagAlternateGroup, a.Reader.GetUInt16());
                         directory.Set(QuickTimeTrackHeaderDirectory.TagVolume, a.Reader.Get16BitFixedPoint());
                         a.Reader.Skip(2L);
-                        directory.Set(QuickTimeTrackHeaderDirectory.TagMatrix, a.Reader.GetInt32Array(9));
+                        directory.Set(QuickTimeTrackHeaderDirectory.TagMatrix, a.Reader.GetMatrix());
```

Comment:
Does this mean `GetInt32Array` is no longer needed? Can you remove it?

Reply:
Removed the function.